### DISCUSSION TITLE
Which pin on Crazyflie Bolt 1.1 corresponds to IOREF?

### DISCUSSION THREAD

**Discussion by Residualstress** [bitcraze/discussions#1045] [Q&A] [answered]
Dear Sir/Madam,

I'm trying to connect a single-board computer with Crazyflie Bolt 1.1. I do this following this picture.
![1](https://github.com/bitcraze/discussions/assets/92587824/fa3af2a3-3406-4758-b389-f8d4ad6480f2)
There is a note of this picture:
Make sure that you do not use voltages in excess of 5.5V on all pins, otherwise you may damage your JeVois unit.
Power consumption on the serial port (all pins combined) is less than 1mA. Hence, it is appropriate to directly use the IOREF pin of any Arduino (which, often, is rated for 50mA or less) with JeVois
IOREF is an input to JeVois, only providing a voltage reference that determines which voltages should be used on the RX and TX pins.
Reference website: http://jevois.org/doc/UserSerial.html

But now, my question is which pin on Crazyflie Bolt corresponds to IOREF? Since IOREF provides a voltage reference that determines which voltages should be used on the RX and TX pins.

Thanks,
Weijie

**Comment by tobbeanton** [comment, marked as answer]
You can use the VCC (3V) on the deck port for this which is the voltage of the system.
